charts/k8ssandra/values.yaml

```diff
@@ -344,65 +345,139 @@ reaper:
+  # secure: false
+
+  # -- Name of the remote storage bucket where backups will be stored. If using 'local' storage, set the
+  # path of the local storage here.
+  bucketName: awstest
+
+  # -- Name of the Kubernetes `Secret` that stores the key file for the
+  # storage provider's API
+  storageSecret: medusa-bucket-key
+
+# Support for accessing the various web-interfaces via K8s ingress controllers.
+# Depending on the ingress available in your cluster enable the appropriate
+# section.
+ingress:
+  traefik:
+    # -- Enable Traefik custom resources as part of this release.
     enabled: false
 
-    # Configures the Medusa image which is built from
-    # https://github.com/thelastpickle/cassandra-medusa/tree/master/k8s.
-    image:
-      # -- Specifies the container repository for Medusa
-      repository: docker.io/k8ssandra/medusa
-      # -- Tag of an image within the specified repository
-      tag: 1e8a4695
-      # -- The image pull policy
-      pullPolicy: IfNotPresent
-
-    # -- Configures the Cassandra user used by Medusa when authentication is
-    # enabled. If neither `cassandraUser.secret` nor `casandraUser.username` are
-    # set, then a Cassandra user and a secret will be created. The username will
-    # be medusa. The secret name will be of the form {clusterName}-medusa. The
-    # password will be a random 20 character password. If `cassandraUser.secret`
-    # is set, then the Cassandra user will be created from the contents of the
-    # secret. If `cassandraUser.secret` is not set and if
-    # `cassandraUser.username` is set, a secret will be generated using the
-    # specified username. The password will be generated as previously
-    # described.
-    cassandraUser:
-      secret: ""
-      username: ""
+    # Repair service
+    repair:
+      # -- Enables Reaper Traefik ingress definitions. Note this will **only**
+      # work if `ingress.traefik.enabled` is also `true`
+      enabled: true
+
+      # -- Traefik entrypoints where traffic is sourced.
+      entrypoints:
+        - web
 
-    # -- Enables usage of a bucket across multiple clusters.
-    multiTenant: false
+      # -- Hostname Traefik should use for routing requests to the repair UI. If
+      # using a local deployment consider leveraging dynamic DNS services like
+      # xip.io. Example: `repair.127.0.0.1.xip.io` will return `127.0.0.1` for
+      # DNS requests routing requests to your local machine.
+      host: repair.k8ssandra.cluster.local
 
-    # -- API interface used by the object store. Supported values include `s3`, 's3_compatible'
-    # and `gcs`. For pod mapped storage, use 'local'
-    storage: s3
+    # Cassandra native transport ingress support
+    cassandra:
+      # -- Enables Cassandra Traefik ingress definitions. Note this will
+      # **only** work if `ingress.traefik.enabled` is also `true`. Additionally,
+      # this is mutually exclusive with
+      # ingress.traefik.stargate.cassandra.enabled
```

Comment:
This is out of date. I'm now thinking that maybe you have more merging to do and that this predates #372 being merged.